### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by tseemann** [andersen-lab/ivar#40]
```

           -t    Minimum fration of files required to contain the same variant. Specify value within [0,1]. (Default: 1)

```
fration :crossed_swords:

**Comment by cmaceves** [comment]
addressed in PR #121

**Comment by gp201** [comment]
![200w (1)](https://user-images.githubusercontent.com/30050862/164791010-83df8d8a-8431-40bb-a957-83279e2966c5.gif)

https://github.com/andersen-lab/ivar/search?q=fration
